service-tests/github.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'github', title: 'Github' });
+module.exports = t;
+
+
+t.create('File size')
+  .get('/size/webcaetano/craft/build/craft.min.js.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('size'),
+    value: Joi.string().regex(/^[0-9]*[.]?[0-9]+\s(B|kB|MB|GB|TB|PB|EB|ZB|YB)$/)
+  }));
```

Comment:
Sorry, I should have explained that better!

The purpose of adding tests is to make sure the `repo or file not found` code branch and the `unknown file` code branch are working as intended. There needs to be a separate test for each one, with an exact match on the string in the response. Otherwise we don't know the API can ever return something that will trigger that branch.

For example, you could add a test for `/size/some-nonexistent-user/some-nonexistent-repo/some-nonexistent-file.md`. I imagine that will hit one of those. Then identify another badge URI which will hit the other.